### PR TITLE
baseweb selector update

### PR DESCRIPTION
**Pull request by gaitho**

### CODE REVIEW

**Comment by gaithoben** [comment]
Pull First. Date Pickers are Already Updated.

**Comment by gaitho** [comment]
Updates are merged in commit e619267

**Comment by gaithoben** [comment]
Please pull and update your components.
LayerManager should import from /baseweb
I dont think there is need to have 2 components for Date Picking. One can work well. but can be re exported with prop range <DatePicker range {...props} />

**Comment by gaithoben** [comment]
https://github.com/cloudhubke/cloudhub-components/blob/4c8c4a53a1b3d9d19ca0afbd9a54d7560ef18309/src/baseweb/BaseWebDaterangePicker.js#L3

This is throwing a warning

let BaseWebDaterangePicker import from BaseWebPicker and return something like <DatePicker range {...props} />

File: src/baseweb/BaseWebSelect.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import isPlainObject from 'lodash/isPlainObject';
+import isEqual from 'lodash/isEqual';
 import { Select, TYPE } from 'baseui/select';
 import { LayersManager } from 'baseui/layer';
```
Comment (gaithoben) [inline src/baseweb/BaseWebSelect.js]: This will throw

Comment (gaithoben) [reply src/baseweb/BaseWebSelect.js]: Replace  import { LayersManager } from 'baseui/layer';   with  import { LayersManager } from './LayersManager';

**Pull request merged**